solenspipe/solenspipe.py

```diff
@@ -27,54 +31,59 @@ def get_mask(lmax=3000,car_deg=2,hp_deg=4,healpix=False,no_mask=False):
             shape,wcs = enmap.fullsky_geometry(res=res)
             mask = enmap.ones(shape,wcs)
         else:
-            afname = f'{opath}/car_mask_lmax_{lmax}_apodized_{car_deg:.1f}_deg.fits'
+            
+            afname = f'{mpath}/car_mask_lmax_{lmax}_apodized_{car_deg:.1f}_deg.fits'
             mask = enmap.read_map(afname)[0]
     return mask
 
 def initialize_args(args):
```

Comment:
I'm not wild about this function - there's a lot of pretty redundant lines like `use_cmblensplus = not(args.flat_sky_norm)`. Can we just have scripts initialize an ``SOLensInterface instance directly, rather than have this intermediate step of manipulating some other similar but slightly different set of arguments to those required for `SOLensInterface`? @qujia7 @msyriac

Reply:
(but let's deal with that in a subsequent pull request anyway)